app/styles/components/sy-checkbox.scss

```diff
@@ -0,0 +1,6 @@
+.sy-checkbox > input[type="checkbox"]:indeterminate + label:after {
+  content: "\2012";
```

Comment:
What is this about?

Reply:
https://www.compart.com/de/unicode/U+2012

Seems to be a `-` sign. Did not add that. This was already there :shrug: